cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PullReplication.java

```diff
@@ -33,8 +33,8 @@
     /**
      * URI for this replication's remote database.
      *
-     * Include username and password in the URL, or supply an Authorization header using
-     * setCustomHeaders() in CouchConfig.
+     * Include username and password in the URL, or supply an Authorization header using an
+     * HttpConnectionRequestInterceptor
```

Comment:
Isn't the `CookieInterceptor`added by default? In which case this statement could be pretty misleading or is it more of an internal note hidden behind the `ReplicatorBuilder`?